### PR TITLE
refactor(delegate): add orderBy param (blocks / voters)

### PR DESCRIPTION
**Pull request by osrn**
added parameters to enable the script to use existing feature in API

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 setuptools.setup(
     name='solar-client',
     description='A simple Python API client for the Solar Blockchain.',
-    version='2.0.0',
+    version='2.0.1',
```
Comment (sleepdefic1t) [inline setup.py:33]: This bit should really be a separate Pr.

**Comment by sleepdefic1t** [comment]
Conflicts should be resolved once version bump is removed.

File: setup.py (hunk 1 above)
Comment (c0nsol3) [reply setup.py:33]: After removing this, please do not do a version bump. Atm the library isn't published anywhere so there's no need to do version bumps. I hope to be able to get the library ready to be published by the eow (depends on my open PRs).

**Review by sleepdefic1t — APPROVED** [review]

**Review by Cactus-15-49 — APPROVED** [review]
